charts/zalenium/values.yaml

```
@@ -147,11 +147,15 @@ ingress:
   tls: false
   # secretName: my-tls-cert # only needed if tls above is true
   hostname: zalenium.foobar.com
+  # context path without the forward slash (/)
+  # leaving this empty would run zalenium in the root context (/)
+  contextpath: /
   annotations:
-    # kubernetes.io/ingress.class: "nginx"
+    kubernetes.io/ingress.class: "nginx"
```

please, leave this line commented as no everyone is using nginx ingress.
```suggestion
    # kubernetes.io/ingress.class: "nginx"
```